Source/Charts/Data/Implementations/Standard/ChartDataEntry.swift

```diff
@@ -104,8 +104,7 @@ extension ChartDataEntry/*: Equatable*/ {
             return true
         }
 
-        return ((data == nil && object.data == nil) || (data?.isEqual(object.data) ?? false))
```

Comment:
I think we shouldn't drop `data` in compare?

Reply:
@jjatie

Reply:
I don't see how it is relevant to how two values are equal or not. If a consumer really needs to know if they are using the same object, the can use `===`

Reply:
From the `Equatable` documentation:
> Exposing nonvalue aspects of Equatable types other than class identity is discouraged